pmlc/conversion/comp_to_llvm/BUILD

```diff
@@ -24,24 +24,26 @@ plaidml_cc_library(
     name = "comp_to_llvm",
     srcs = [
         "common.cc",
-        "comp_to_ocl.cc",
+#        "comp_to_ocl.cc",
```

Comment:
I think we need a separate library for comp to vulkan.

Reply:
fixed